lib/bloc/pos_catalog/bloc.dart

```diff
@@ -176,6 +180,16 @@ class PosCatalogBloc with AsyncActionsHandler {
     });
   }
 
+  void _trackDefaultSaleNote(Stream<BreezUserModel> userStream) {
+    userStream.listen((user) {
+      final defaultSaleNote = user.defaultPosNote;
+      final currentSale = _currentSaleController.value;
+      _currentSaleController.add(currentSale.copyWith(
```

Comment:
just want to make sure that we don't override existing sale models. Where is the point that the _currentSaleController cleans up existing sales?

Reply:
It seems to be here: https://github.com/breez/breezmobile/blob/master/lib/bloc/pos_catalog/bloc.dart#L150

Reply:
@ademar111190 what happens if the sale is not paid (e.g cancelled)? does the controller still hold the old value?

Reply:
If I cancel the value is kept, but if I clear the sale the value is set to empty 🤔 
I guess it should be "cleared" to the default right?

Reply:
I did apply this change, now the sale has the default note after a clear.